python/ray/tune/syncer.py

```diff
@@ -301,11 +302,11 @@ def close(self):
         pass
 
 
-class _DefaultSyncer(Syncer):
-    """Default syncer between local storage and remote URI."""
+class _BackgroundSyncer(Syncer):
```

Comment:
Note for review: These changes are not necessary for the example CustomCommandSyncer to work, but they provide a better abstraction if we want to move it to async later. It's a small internal refactoring.